internal/commands/result.go

```diff
@@ -1027,6 +1031,17 @@ func parseLocationKics(results *wrappers.ScanResult) wrappers.SonarLocation {
 	auxLocation.TextRange = auxTextRange
 	return auxLocation
 }
+func parseLocationSca(results *wrappers.ScanResult) wrappers.SonarLocation {
+	var auxLocation wrappers.SonarLocation
+	auxLocation.FilePath = strings.TrimLeft(results.ScanResultData.Filename, "/")
```

Comment:
Hi @margaritalm  I believe this scanREsultsData.Filename is not populated for SCA results. You need to use result.ScanResultData.ScaPackageCollection.Locations. Can you please check a SCA result ? 

Thanks

Reply:
We create this method for Sarif results : parseSarifResultsSca